src/next/index.ts

```diff
@@ -163,6 +164,42 @@ export function sendOperationError(operationId: string, operationName: string, e
     sendEvent(event);
 }
 
+/**
+ * Instrument callback for a procedure (regarded as a step in an operation).
+ * @param operationId A unique identifier for the operation to which the step belongs.
+ * @param stepName Name of the step.
+ * @param cb The callback function with a unique Id passed by its 1st parameter.
+ * @returns The instrumented callback.
+ */
+export async function instrumentOperationStep(
```

Comment:
this method return a function, right? so why is it  async?

Reply:
It returns a promise-like thing.